eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/GA4GHVariantCallSetWSServer.java

```diff
@@ -61,7 +69,7 @@ public GA4GHVariantCallSetWSServer() { }
      * @see http://ga4gh.org/documentation/api/v0.5/ga4gh_api.html#/schema/org.ga4gh.GASearchCallSetsRequest
      */
     @RequestMapping(value = "/search", method = RequestMethod.GET)
-    public GASearchCallSetsResponse getCallSets(@RequestParam("variantSetIds") String files,
+    public GASearchCallSetsResponse getCallSets(@RequestParam("variantSetIds") List<String> files,
                                                 @RequestParam(name = "pageToken", required = false) String pageToken,
                                                 @RequestParam(name = "pageSize", defaultValue = "10") int limit,
```

Comment:
GA4GH endpoints should use `Pageable` objects and remove the histogram functionality, in a similar fashion to the rest of the API.